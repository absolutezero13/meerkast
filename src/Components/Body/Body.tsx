import React from "react";
import "./Body.css";
import { Link } from "react-router-dom";
const Body: React.FC = () => {
  return (
    <div className="body">
      <Link to="/movies">
        <div className=" body__item">
          <p>MOVIES</p>
        </div>
      </Link>
      <Link to="/series">
        <div className=" body__item">
          <p>SERIES </p>
          <img src="../../../images/placeholder.png" alt="" />
        </div>
      </Link>
    </div>
  );
};

export default Body;
