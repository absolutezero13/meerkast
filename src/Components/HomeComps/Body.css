.body {
  display: flex;
  height: 60vh;
  padding: 2% 10%;
  grid-gap: 10px;
  justify-content: center;
  font-weight: 900;
}

.body__item {
  cursor: pointer;
  font-size: 2rem;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 35vh;
  width: 15rem;
  color: white;
  border: 1px solid black;
  text-align: center;
  background: rgba(30, 30, 30, 255);
  background-image: url("../../images/placeholder.png");
  background-repeat: no-repeat;
  background-size: contain;
}

a {
  text-decoration: none;
  height: 35vh;
  width: 15rem;
}
