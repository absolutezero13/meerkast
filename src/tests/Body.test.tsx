import React from "react";
import { render } from "@testing-library/react";
import Body from "../Components/Body/Body";
import { BrowserRouter } from "react-router-dom";

const { container } = render(
  <BrowserRouter>
    <Body />
  </BrowserRouter>
);

// SnapShot Test
it("body matches snapshot", () => {
  expect(container).toMatchSnapshot();
});
