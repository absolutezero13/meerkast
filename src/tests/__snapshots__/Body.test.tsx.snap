// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`body matches snapshot 1`] = `
<div>
  <div
    class="body"
  >
    <a
      href="/movies"
    >
      <div
        class=" body__item"
      >
        <p>
          MOVIES
        </p>
      </div>
    </a>
    <a
      href="/series"
    >
      <div
        class=" body__item"
      >
        <p>
          SERIES 
        </p>
        <img
          alt=""
          src="../../../images/placeholder.png"
        />
      </div>
    </a>
  </div>
</div>
`;
